jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/exception/ResourceTemplateUpdateException.java

```diff
@@ -26,4 +29,31 @@ public ResourceTemplateUpdateException(final String entityName, final String res
         super(resourceName + " of " + entityName + " update failed!", t);
     }
 
+
+    /**
+     * Exception wrapper for resource template update failures.
+     *
+     * @param application
+     * @param group
+     */
+    public ResourceTemplateUpdateException(final JpaApplication application, final JpaGroup group) {
+        super("Resource update for group " + group.getName() + " of application " + application.getName() + " failed.");
```

Comment:
Use MessageFormat.format() instead. 
It can be a performance hit (which we're not really concerned about in this case), but I think it's easier to read and it will make it easier to internationalize when the time comes.